cmd/version.go

```diff
@@ -7,24 +7,68 @@ package cmd
 import (
 	"fmt"
 
+	"io"
+	"os"
+
 	"github.com/spf13/cobra"
 
+	"github.com/open-policy-agent/opa/internal/report"
+
+	"context"
+	"crypto/rand"
+
+	"github.com/open-policy-agent/opa/internal/uuid"
 	"github.com/open-policy-agent/opa/version"
 )
 
 func init() {
 
+	var check bool
 	var versionCommand = &cobra.Command{
 		Use:   "version",
 		Short: "Print the version of OPA",
 		Long:  "Show version and build information for OPA.",
 		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Println("Version: " + version.Version)
-			fmt.Println("Build Commit: " + version.Vcs)
-			fmt.Println("Build Timestamp: " + version.Timestamp)
-			fmt.Println("Build Hostname: " + version.Hostname)
+			generateCmdOutput(os.Stdout, check)
 		},
 	}
 
+	versionCommand.Flags().BoolVarP(&check, "check", "c", false, "check for latest OPA release")
```

Comment:
Might be worth adding a comment here that just reminds is that some tools may use `opa version` to inspect what version if available for feature flagging purposes. This means that we probably want to keep this off by default.

Reply:
Added a comment about this.